ipatests/test_xmlrpc/test_krbtpolicy.py

```diff
@@ -116,21 +116,407 @@ class test_krbtpolicy(Declarative):
+                     ),
+            expected=errors.ConversionError(
+                name='pkinit_maxrenew', error='must be an integer'),
+        ),
+        dict(
+            desc='Try updating invalid value auth indicator radius',
+            command=('krbtpolicy_mod', [user1],
+                     dict(krbauthindmaxticketlife_radius=u'abc123')
+                     ),
+            expected=errors.ConversionError(
+                name='radius_maxlife', error='must be an integer'),
+        ),
+        dict(
+            desc='Try updating max value auth indicator radius',
+            command=('krbtpolicy_mod', [user1],
+                     dict(krbauthindmaxticketlife_radius=u'2147483648')
+                     ),
+            expected=errors.ValidationError(
+                name='radius_maxlife', error='can be at most 2147483647'),
+        ),
+        dict(
+            desc='Try updating min value auth indicator radius',
+            command=('krbtpolicy_mod', [user1],
+                     dict(krbauthindmaxticketlife_radius=u'0')
+                     ),
+            expected=errors.ValidationError(
+                name='radius_maxlife', error='must be at least 1'),
+        ),
+        dict(
+            desc='Try updating negative value auth indicator radius',
+            command=('krbtpolicy_mod', [user1],
+                     dict(krbauthindmaxticketlife_radius=u'-1')
+                     ),
+            expected=errors.ValidationError(
+                name='radius_maxlife', error='must be at least 1'),
+        ),
+        dict(
+            desc='Try updating negative value auth indicator radius',
+            command=('krbtpolicy_mod', [user1],
+                     dict(krbauthindmaxrenewableage_radius=u'-1')
+                     ),
+            expected=errors.ValidationError(
+                name='radius_maxrenew', error='must be at least 1'),
+        ),
+        dict(
+            desc='Try updating min value auth indicator radius',
+            command=('krbtpolicy_mod', [user1],
+                     dict(krbauthindmaxrenewableage_radius=u'0')
+                     ),
+            expected=errors.ValidationError(
+                name='radius_maxrenew', error='must be at least 1'),
+        ),
+        dict(
+            desc='Try updating max value auth indicator radius',
+            command=('krbtpolicy_mod', [user1],
+                     dict(krbauthindmaxrenewableage_radius=u'2147483648')
+                     ),
+            expected=errors.ValidationError(
+                name='radius_maxrenew', error='can be at most 2147483647'),
+        ),
+        dict(
+            desc='Try updating invalid value auth indicator radius',
+            command=('krbtpolicy_mod', [user1],
+                     dict(krbauthindmaxrenewableage_radius=u'abc123')
+                     ),
+            expected=errors.ConversionError(
+                name='radius_maxrenew', error='must be an integer'),
+        ),
```

Comment:
The above code (tests for invalid args) can be simplified with a double loop similar to:
```
invalid_values = [ (u'abc123', 'must be an integer'), (u'2147483648', 'can be at most 2147483647'), (u'0', 'must be at least 1'), (u'-1', 'must be at least 1')]
parameters = [ ('krbauthindmaxrenewableage_radius', 'radius_maxrenew'), ('krbauthindmaxticketlife_radius', 'radius_maxlife'), ...]
for (value,error) in invalid_values:
  for (param, param_name) in parameters:
    tests.append(create_dict(desc='Try updating invalid %s'.format(param_name),param=param, value=value,name=param_name, error=error)
```
And you need to create a function
```
def create_dict(desc, param, param_name, value, error):
    cmd_args=dict()
    cmd_args[param]=value
    return dict(desc=desc, command=('krbtpolicy_mod', [user1],cmd_args),expected=errors.ConversionError(name=name, error=error))
```